ale/drivers/nh_drivers.py

```diff
@@ -99,6 +99,10 @@ def sensor_name(self):
     def frame_chain(self):
         self._props['exact_ck_times'] = False
         return super().frame_chain
+    
+    @property
+    def sensor_model_version(self):
+        return 2
```

Comment:
are there any existing tests for the cam model version stuff that we need to add or update?

Reply:
I will double check